alvr/client_openxr/src/extra_extensions/body_tracking_fb.rs

```diff
@@ -35,10 +38,27 @@ pub struct BodyTrackerFB {
     ext_fns: raw::BodyTrackingFB,
 }
 
+#[repr(C)]
+struct SystemPropertiesBodyTrackingFidelityMETA {
+    ty: xr::StructureType,
+    next: *mut std::ffi::c_void,
+    supports_body_tracking_fidelity: sys::Bool32,
+}
+
+#[repr(C)]
+enum BodyTrackingFidelityMode {
+    Low = 1,
+    High = 2,
+}
+
+type RequestBodyTrackingFidelityMETA =
+    unsafe extern "system" fn(sys::BodyTrackerFB, BodyTrackingFidelityMode) -> sys::Result;
+
 impl BodyTrackerFB {
     pub fn new<G>(
         session: &xr::Session<G>,
         body_joint_set: xr::BodyJointSetFB,
+        system: xr::SystemId,
```

Comment:
Ordering the parameters, i would put `system` just under session. Also, you should have high fidelity as a setting, and passed as parameter here. In case high fidelity is not available you can gracefully fall back to low accuracy. The setting should be `prefer_high_fidelty`, and put under `prefer_full_body`.